README.MD

```diff
@@ -1,38 +1,43 @@
 # Nadel: A distributed GraphQL engine
 
-Nadel is a Java library to combine several graphql services together.
-This is achieved by combining several underlying GraphQL services (schemas) into one schema (overall schema).
+Nadel is a Kotlin library to combine several GraphQL services together. This is achieved by combining several underlying
```

Comment:
Mention it supports consumption via Java, right?

Reply:
Yup, done.